src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -771,63 +811,69 @@ void testUpdateDatasetDataUse_NotModified() {
     when(datasetService.findDatasetById(any())).thenReturn(d);
 
     initResource();
-    Response response = resource.updateDatasetDataUse(new AuthUser(), 1, du.toString());
-    assertEquals(HttpStatusCodes.STATUS_CODE_NOT_MODIFIED, response.getStatus());
+    try (var response = resource.updateDatasetDataUse(new AuthUser(), 1, du.toString())) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_NOT_MODIFIED, response.getStatus());
+    }
   }
 
   @Test
   void testFindAllDatasetsStreaming() throws Exception {
     var dataset = new Dataset();
-    dataset.setDatasetId(RandomUtils.nextInt(100, 1000));
+    dataset.setDatasetId(randomInt(100, 1000));
     when(userService.findUserByEmail(any())).thenReturn(user);
     final Gson gson = GsonUtil.buildGson();
     StreamingOutput output = out -> out.write(gson.toJson(List.of(dataset)).getBytes());
     when(datasetService.findAllDatasetsAsStreamingOutput()).thenReturn(output);
     initResource();
 
-    Response response = resource.findAllDatasetsStreaming(authUser);
-    assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
-    var entity = (StreamingOutput) response.getEntity();
-    var baos = new ByteArrayOutputStream();
-    entity.write(baos);
-    var entityString = baos.toString();
-    Type listOfDatasetsType = new TypeToken<List<Dataset>>() {
-    }.getType();
-    List<Dataset> returnedDatasets = gson.fromJson(entityString, listOfDatasetsType);
-    assertThat(returnedDatasets, hasSize(1));
-    assertEquals(dataset.getDatasetId(), returnedDatasets.get(0).getDatasetId());
+    try (var response = resource.findAllDatasetsStreaming(authUser)) {
+      assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
+      var entity = (StreamingOutput) response.getEntity();
+      var baos = new ByteArrayOutputStream();
+      entity.write(baos);
+      var entityString = baos.toString();
+      Type listOfDatasetsType = new TypeToken<List<Dataset>>() {
+      }.getType();
+      List<Dataset> returnedDatasets = gson.fromJson(entityString, listOfDatasetsType);
```

Comment:
The extra assignment isn't needed and using `TypeToken` as a parameter lets the compiler use type inference.
```suggestion
      List<Dataset> returnedDatasets = gson.fromJson(entityString, new TypeToken<>() {});
```